modules/kubernetes/pages/k8s-operator/backup-and-restore.adoc

```diff
@@ -0,0 +1,710 @@
+= Backup and Restore
+
+The Backup and Restore functionality of the Kubernetes operator allows for incremental and scheduled backups as well as detailed logging.
+
+Backup and restore jobs work by internally creating TigerGraphBackup, TigerGraphRestore, and TigerGraphScheduledBackup functions that carry out jobs according to the user settings.
+This concept is important for understanding the relationship between the console commands you enter and the results from Kubernetes.
+
+== Prerequisites
+
+You must have the TigerGraph Kubectl Plugin installed and a working cluster available.
+
+
+The `kubectl tg backup` command requires `kubectl`, `helm`, `jq`, and `yq` to be installed.
+
+== Backup command syntax and options
+
+[source.wrap,bash]
+----
+kubectl tg backup [create|update] [OPTIONS]
+
+Options:
+  -h|--help: show this message
+  -n|--namespace :              set namespace to deploy TG cluster, default namespace is current namespace
+  -c|--cluster-name :           (required)set cluster-name to deploy TG cluster
+  --tag :                       (required)specify the tag of backup files. e.g. if you specify --tag daily, the backup file will be daily-20xx-xx-xxTxxxxxx
+  --staging-path :              specify where to store the temporary files
+  --timeout :                   the backup timeout in seconds,default: 18000
+  --compress-process-number :   the number of concurrent process for compression during backup
+                                value 0 means the number of processes used to compress equals
+                                the number of the node's CPU cores. And the default value is 0
+  --incremental :               perform incremental backup
+  --full :                      perform full backup
+  --destination :               set the destination to store backup files, support local and s3 now
+  Followings are about the configuration of different destination:
+  If destination is local,you should provide:
+    --local-path :              set the local path where to store backup files
+  If destination is s3:
+    --s3-bucket :               S3 Bucket name
+    --aws-access-key-id :       the aws access key id
+    --aws-secret-access-key :   the aws secret access key
```

Comment:
option --aws-access-key-id and aws-secret-access-key are removed. Replace them with --aws-secret. Please refer to https://graphsql.atlassian.net/wiki/spaces/infra/pages/2946171832/K8S+Backup+Restore#create-and-update-backup